docs/source/AdministratorGuide/Systems/Transformation/index.rst

```diff
@@ -254,6 +257,15 @@ In order to use the ByJobType plugin, one has to:
           AutoAddedSites += LCG.RAL.uk
           AutoAddedSites += LCG.SARA.nl
           AutoAddedSites += LCG.RRCKI.ru
+          DataReconstruction
+          {
+            Exclude = ALL
+            Exclude += LCG.CERN.cern
```

Comment:
Yes, you may say that ;-)
The alternative would be to include within the jobType section another set of AutoAddedSites that would supersede the general one. Would you prefer that?